### DISCUSSION TITLE
Help for publication

### DISCUSSION THREAD

**Discussion by OlliSant** [a-cosmology-group/acg#151] [Metaphysics and Philosophy]
I updated my thoughts, my Essay, and model http://www.santavuori.com/Galakseja3a.pdf Where can I try to publish it? In some Journal, which one can I try? Comments welcome too.

**Comment by bgreenman** [comment]
I wish I knew, but I don't know what refereed journal or reputable academic publisher would publish alternative ideas in science

**Comment by RedshiftDrift** [comment]
Usually one can submit to a journal cited in the references, but you cite only one journal (indirectly), QSSC which was published in J. Astrophys. Astron.  This journal would not publish a philosophical essay such as yours.

The book you cite: "An introduction to the philosophy of science" (1953) must have references in journals that still exist today?  Otherwise I'd say your list of references is incomplete.  You cannot cite the ACG-website, this is not a paper archive.

You have to find the original papers as published in the litterature and cite those papers directly.

**Reply by OlliSant** [reply]
That is a good advise. From the wiki articles I can block the primary sources. And philosophy seems the only alternative. They are also mostly BB- thinkers.

It also means that ACG should work for a Journal in cosmology, for alternative theories.

**Reply by RedshiftDrift** [reply]
It is a lot of work to publish a Journal, and if it's not mainstream the readership remains very limited.

"Physics Essays" edited by Emilio Panarella may publish your paper.  Unfortunately it has a bad reputation and your paper won't be widely read.
"Apeiron" by Roy Keys may also publish your paper.  However the last publication dates from 2018.

**Reply by OlliSant** [reply]
Maybe it's best to be just here in ACGs papers and in the home page. Some Journals asked from me an article, I don't know why, and I writed this one and did send there the previous version which is in the models here, which Louis collected. And I informed them that new one comes. It's maybe a commercial thing and I must pay for publication. I don't know yet.

**Comment by OlliSant** [comment]
And somebody who agrees with me, might write this article better with me? And make studies with these hypothesis, or form a physical- mathematical theory with these premises? Or a group of scholars.

**Comment by clmarchesan** [comment]
I have many open journals that write to me too, but they are all paid and expensive. I have always written on viXra, and lately, I have built a website https://www.4-sphere-cosmology.eu/ mainly because I update it too frequently. It has some impressions but very few clicks. Besides ACG, I believe viXra remains if you limit the updates.

**Reply by OlliSant** [reply]
Could you write my article better and then we send it together to viXra?

**Comment by clmarchesan** [comment]
However, if your site already has many clicks you don't need viXra. Have you seen the publications on Amazon?

**Reply by OlliSant** [reply]
What publications in Amazon? My website has 34.000 clicks.

**Comment by bgreenman** [comment]
It seems that despite internet technology, publishing is still hard to do, if you want a larger scholarly  audience

**Reply by OlliSant** [reply]
Yes, and in media too.

**Comment by Francois-Zinserling** [comment]
viXra was an answer against arxiv.org, because posting onto the arxiv meant you had to belong to an elitist group, or find a benefactor that considers your work 'worthy'. For us fringe scientists, a pureblood physicist would not risk his career by nominating e.g. me as an arxiv candidate. What viXra is doing is noble, but it is only a storage archive, it doesn't engage thought or discussion. That said, yes, certainly you should post on viXra so that your work is archived and registered as yours.

Publishing is very hard to do. Whatever you do, don't pay for a publication, because 99.9% of those are just there to rob you. There are better ways of getting your work 'out there'. Other journals will make you sweat to make your whole article fit their formats, and then still decline to publish. Not saying you shouldn't pursue it, just that it is very very tough.

If you are more interested in engaging with peers, then this is one place to be (ACG=cosmology related), or else you can try platforms like ResearchGate, where your uploaded work is considered published, but not peer reviewed. You can entice reviews from other like-minded thinkers, but bear in mind others are also there to peddle their own work.

**Reply by OlliSant** [reply]
What I want, is that scholars know my ideas. I'm content with my homepage, because there are the dates, when I have invented my specific ideas, inventions, if it happens, that they are important in cosmology. Publication makes some status, but that is not so important. Homepage and ACGs Essays is enough, I wait what happens, and hope all of you much success too in future! I just send articles sometimes somewhere and tell in the homepage and in Internet discussion forums, what happened. I don´t pay anything for publication. A sponsor could be important for whole ACG too. I am a philosophical writer and it stays so. I am not an academical scientist.

**Reply by ExpEarth** [reply]
@Francois-Zinserling It's not quite correct to say you should never pay.  There are quite a few journals now that have a different publication model.  These are titles like _Galaxies_, _Universe_ and all the _Frontiers_ titles, _eg_ _Frontiers in Physics_.  They are peer-reviewed journals where the author or institution normally has to pay a fee, of maybe $1,000 - $2,000.  It's not cheap!  However, these journals are also open access, which means everyone can read them.  To have your articles appear as open access in other academic journals you (or your institution) would still need to pay a similar amount.  The disadvantage for many of us here is that we can't have institutions pick up the charge.  But it could still be worthwhile to pay it yourself, if you want your work to get published.  I would say the most important thing is that the journal be peer reviewed.

**Reply by bgreenman** [reply]
The best alternatives so far are ResearchGate and nominal fee open access peer-reviewed journals

**Reply by Francois-Zinserling** [reply]
@ExpEarth yes you are correct, and I apologise for the misrepresentation. The journals you refer to are not trying to rob people!

However, there are just way too many predatory journals around right now and I do wish to advise extreme caution before paying for a publication.

**Comment by clmarchesan** [comment]
34,000 clicks are a real asset in terms of ranking. With viXra, you certainly won't achieve this level of visibility. Excluding paid archives, the only alternative in my opinion would be to directly approach some universities, but I wouldn't know how to assist you with that. Let's keep this discussion open; it's possible that someone from ACG University may come up with some ideas.

**Comment by clmarchesan** [comment]
Perhaps there are better options, but as for me, I discarded the idea of contacting a University directly, I hoped to achieve good visibility, encourage email exchanges and then wait and see what would happen. 


Regarding the costs of peer review, even if many are  against AI, in my speculation I wrote like this:


THIS SPECULATION WITH CHATGPT

I expect a future where every speculation continues to find its way to always be published. There, I would like the paper to be accompanied by a report card that evaluates quality of scientific content. Provided, at the discretion, by artificial intelligence (AI), it could help the reader to choose among more papers.
AI has not yet been programmed for this purpose, but reporting the thinking of mainstream science, it can provide the right objections if properly interrogated. Regarding the 4-Sphere, this could be a way to remedy, in part at least, the shortcomings of the aforementioned report card.
Accordingly, to know the thought of AI, we have tried to engage in the discussion ChatGPT, our Artificial Intelligence (see https://openai.com/). The purpose is not just to fill the gap in knowledge but also, through conflicting opinions, to get some form of supervision.

**Reply by bgreenman** [reply]
It could be a big help, but you still need a human in the loop

**Reply by clmarchesan** [reply]
ChatGPT also comes to a similar conclusion. However, even current peer review is certainly not perfect. This would only be an evaluation and in the end the reader's opinion should still prevail.

**Reply by bgreenman** [reply]
Nothing is perfect and original opinions should be expressed

**Comment by OlliSant** [comment]
We could also make groupings in this revolution against BB. And make models together. Where somebodies are of the same opinion. 
- BB is not good for a theory of the whole universe, it is a theory of the local universe, and not perfect there either 
- gravitation is first but there are other forces too
- 4D is maybe better
- universe is eternal
- universe is infinite, but only boundless, it might have a size.
Or something else common, and then make a better theory together and test it in astronomy.  What is needed is corrections to the GR. So very big and difficult process for astronomers. In philosophy this is easier.

**Reply by bgreenman** [reply]
A good starting point
